python-api/gstswitch/server.py

```diff
@@ -22,6 +26,110 @@
 TOOLS_DIR = '/'.join(os.getcwd().split('/')[:-1]) + '/tools/'
 
 
+class ProcessMonitor(subprocess.Popen):
+    """Runs a Command in a Background-Thread and monitors it's output
```

Comment:
English capitalization rules are a nightmare for a non native-speaker :) but - fixed (in this place).